cmd/openshift-install/log.go

```diff
@@ -56,6 +57,13 @@ func (h *fileHook) Fire(entry *logrus.Entry) error {
 		msgs = strings.Split(orig, "\n")
 	}
 
+	level, err := logrus.ParseLevel(h.config.Level)
```

Comment:
Check err?

Reply:
I am building a "if error then ignore"  kind of system so I don't want this to report anything if error unless maybe a debug line